Documentation/dev-guide/interacting_v3.md

```diff
@@ -267,14 +314,29 @@ bar
 PUT
 foo
 bar_new
+```
 
+```bash
 # watch for changes on key `foo` since revision 3
 $ etcdctl watch --rev=3 foo
 PUT
 foo
 bar_new
 ```
 
+Here is an example to watch only from the last historical change:
+
+```bash
```

Comment:
@xiang90 i have done the change for the same. Let me know if its fine or i need to add some more clarity to it.
Thank you!